### ISSUE TITLE
Add [LxgwWenKai]

### ISSUE THREAD

**Issue by Yuiinars** [google/fonts#4601]
**Font Project Git Repo URL:**
https://github.com/lxgw/LxgwWenKai

**Super short description of the Font Family:**
An open-source Chinese font derived from Fontworks' Klee One.

**Requirements:**

I understand that Google Fonts will publish only fonts that matches its requirements, and I can confirm the project meets them (by ticking the cases, or putting x between the square brackets in text mode):

- [x] The entire font project is licensed under the [OFL](https://scripts.sil.org/cms/scripts/page.php?site_id=nrsi&id=OFL)
- [x] There is no "Reserved Font Name" mention in the OFL license
- [x] Family name is unique according to namecheck.fontdata.com
- [x] The font supports at least [GF Latin Core glyphset](https://github.com/googlefonts/glyphsets/tree/main/GF_glyphsets/Latin/)
- [x] The repository respect the [upstream repo structure](https://github.com/googlefonts/gf-docs/tree/main/Spec#upstream-repo-structure)
- [x] The source files are available in the repository
- [x] I read, agree and comply with the full [contributing requirements](https://github.com/google/fonts/blob/main/CONTRIBUTING.md)
- [ ] I am the author of the font or the author agreed to this submission
- [ ] I will maintain the repository

**Comment by lxgw** [comment]
I'd prefer to submit [the lite edition](https://github.com/lxgw/LxgwWenKai-Lite) to Google Fonts...

**Comment by davelab6** [comment]
@lxgw this is great work! I was not able to find the source files and build scripts to your OFL project repos, are they available?

Since the GF API uses Unicode Range subsetting, I think we ought to take the full project, not the lite one - but please correct me if I'm wrong that the 'lite' version is lighter in terms of smaller glyph set.

**Comment by NightFurySL2001** [comment]
> Since the GF API uses Unicode Range subsetting, I think we ought to take the full project, not the lite one - but please correct me if I'm wrong that the 'lite' version is lighter in terms of smaller glyph set.

The full project contains glyphs that were generated using `zi2zi` which is of poor quality, although covering CJK Unified Ideogrpahs and Extension A completely. The Lite version which is smaller in glyph set only contains hand made glyphs and characters that are generally used in Simplified and Traditional Chinese, and no hangul characters.

**Comment by NightFurySL2001** [comment]
@RosaWagner @davelab6 Sources are uploaded to both repo with build script and modified to match upstream repo structure. I will help @lxgw to maintain the repository if needed.

**Comment by lxgw** [comment]
> Since the GF API uses Unicode Range subsetting, I think we ought to take the full project, not the lite one - but please correct me if I'm wrong that the 'lite' version is lighter in terms of smaller glyph set.

Based on Klee One, the full edition contains all the characters of Basic CJK Unified Ideographs, several CJK Unified Ideograph Extensions (from Ext.A to Ext.H draft) and some Hangul Syllables in common use, while the lite edition contains characters commonly used in Simplified and Traditional Chinese and no Hangul characters.

But some glyphs in the full edition were generated by zi2zi-pytorch that the quality of them might be more terrible. The lite edition contains handmade glyphs only.

> I was not able to find the source files and build scripts to your OFL project repos, are they available?

With the help of @NightFurySL2001, now the sources files and build scripts of both editions have been uploaded to their respective repositories.

**Comment by lxgw** [comment]
> > Since the GF API uses Unicode Range subsetting, I think we ought to take the full project, not the lite one - but please correct me if I'm wrong that the 'lite' version is lighter in terms of smaller glyph set.
> 
> Based on [Klee One](https://github.com/fontworks-fonts/Klee), the [full edition](https://github.com/lxgw/LxgwWenKai) contains all the characters of Basic CJK Unified Ideographs, several CJK Unified Ideograph Extensions (from Ext.A to Ext.H draft) and some Hangul Syllables in common use, while the [lite edition](https://github.com/lxgw/LxgwWenKai-Lite) contains characters commonly used in Simplified and Traditional Chinese and no Hangul characters.
> 
> But some glyphs in the full edition were generated by [zi2zi-pytorch](https://github.com/EuphoriaYan/zi2zi-pytorch) that the quality of them might be more terrible. The lite edition contains handmade glyphs only.
> 
> > I was not able to find the source files and build scripts to your OFL project repos, are they available?
> 
> With the help of @NightFurySL2001, now the sources files and build scripts of **both editions** have been uploaded to their respective repositories, and both repositories have been modified to meet the [upstream repo structure](https://github.com/googlefonts/gf-docs/tree/main/Spec#upstream-repo-structure).

**Comment by lxgw** [comment]
The Traditional Chinese version [LXGW WenKai TC](https://github.com/lxgw/LxgwWenkaiTC) has been updated to v0.9 and the repo has been modified to meet [upstream repo structure](https://github.com/googlefonts/gf-docs/tree/main/Spec#upstream-repo-structure). May I open another issue to submit this font?

**Comment by medicalwei** [comment]
Note that this font has RFN "霞鹜" and "LXGW", and I am asking about it in https://github.com/lxgw/LxgwWenKai/issues/80.

**Comment by Honoka55** [comment]
I’d kindly request a reconsideration regarding the font’s copyright status in light of concerns raised in my [earlier issue](https://github.com/lxgw/LxgwWenKai/issues/128). Need further clarification whether the Lite version is also involved in the copyright issue.

**Comment by aaronbell** [comment]
I believe that this variant can be created as a subsetted version of the existing LXGW WenKai project, and can be created as part of the same build system. Unless there's something else going on aside from simply subsetting?

**Comment by NightFurySL2001** [comment]
Yes, WenKai Lite is just a subsetted version of WenKai (similarly in GB). I actually updated the upstream build system on his local machine after the build system PR breakage in TC and written a program that starts from Lite, then merge in additional characters from another UFO to form the standard version. The only request is probably requiring both version in their own separate repo to prevent mixing (it's 6 font files per project).
